mle/cli.py

```diff
@@ -113,29 +112,47 @@ def report(ctx, repo, model, user, visualize):
 
 @cli.command()
 @click.option('--model', default=None, help='The model to use for the chat.')
-def kaggle(model):
+@click.option('--auto', is_flag=True, help='Use auto mode to generate the coding plan.')
+@click.option('--description', default=None, help='The description of the competition.')
+@click.option('--datasets', default=None, help='The .csv dataset home to use for the competition.')
+@click.option('--submission', default='./submission.csv', help='the path of the kaggle submission .csv file.')
+@click.option('--sub_example', default=None, help='the path to the kaggle submission example .csv file.')
+@click.option('--comp_id', default=None, help='the kaggle competition id.')
+def kaggle(model, auto, description=None, datasets=None, sub_example=None, submission='.', comp_id=None):
     """
     kaggle: kaggle competition workflow.
+
+    Example command for auto mode:
+    mle kaggle --auto --datasets "/Users/huangyz0918/desktop/spaceship-titanic/prepared/public/train.csv,/Users/huangyz0918/desktop/spaceship-titanic/prepared/public/test.csv"
+     --description "/Users/huangyz0918/desktop/spaceship-titanic/prepared/public/description.md" --submission "./submission.csv"
+     --sub_example "/Users/huangyz0918/desktop/spaceship-titanic/prepared/public/sample_submission.csv" --comp_id "spaceship-titanic"
```

Comment:
tremendous example

Reply:
please update the readme section also

otherwise, people can not enjoy this function